src/app/shared/services/tab.service.ts

```diff
@@ -2,15 +2,18 @@ import { Injectable } from '@angular/core';
 import { Observable, ReplaySubject, Subject } from 'rxjs';
 import { ReportData } from '../interfaces/report-data';
 import { CompareData } from '../../compare/compare-data';
+import { CloseTab } from '../interfaces/close-tab';
 
 @Injectable({
   providedIn: 'root',
 })
 export class TabService {
   private openReportInTabSubject: Subject<ReportData> = new ReplaySubject();
-  openReportInTabObservable: Observable<ReportData> = this.openReportInTabSubject.asObservable();
+  openReportInTab$: Observable<ReportData> = this.openReportInTabSubject.asObservable();
   private openInCompareSubject: Subject<CompareData> = new ReplaySubject();
-  openInCompareObservable: Observable<CompareData> = this.openInCompareSubject.asObservable();
+  openInCompare$: Observable<CompareData> = this.openInCompareSubject.asObservable();
+  private closeTabSubject: Subject<CloseTab> = new ReplaySubject();
+  closeTab$: Observable<CloseTab> = this.closeTabSubject.asObservable();
```

Comment:
Like @philipsens told me in the PR for the [filter.service](https://github.com/wearefrank/ladybug-frontend/blob/master/src/app/debug/filter-side-drawer/filter.service.ts), it would be better to put the private variables separate from the public ones.

Reply:
fixed